src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -202,25 +202,11 @@
 		A list of analyzer suppressions to include in every generated class.
 		This is required for roslyn analyzers that do not support the GeneratedCodeAttribute.
 		-->
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV0056" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV0058" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV1003" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV0085" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV2001" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV2003" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV2004" />
-		<XamlGeneratorAnalyzerSuppressions Include="nventive.Usage-NV2005" />
 
 		<XamlGeneratorAnalyzerSuppressions Include="csharp-618 // Ignore obsolete members warnings" />
 		<XamlGeneratorAnalyzerSuppressions Include="csharp-105 // Ignore duplicate namespaces" />
 		<XamlGeneratorAnalyzerSuppressions Include="csharp-1591 // Ignore missing XML comment warnings" />
 		<XamlGeneratorAnalyzerSuppressions Include="csharp-CS8669 // Ignore annotation for nullable reference types" />
-
-		<!--
-		Ignore disposable fields, Uno manages those on its own.
-		https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1001
-		-->
-		<XamlGeneratorAnalyzerSuppressions Include="dotnet-CA1001" />
```

Comment:
I think we should keep this one?

Reply:
Nope. Not needed. The analyzer does not report diagnostics on generated code, and Roslyn respects the auto-generated comment that we added at some point.

https://github.com/dotnet/roslyn-analyzers/blob/81d9b5d58e3d384ed4a91b75f4a1ece6352e76d9/src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/TypesThatOwnDisposableFieldsShouldBeDisposable.cs#L41